src/functions/module_array.ts

```diff
@@ -82,6 +82,62 @@ export const FLATTEN: AddFunctionDescription = {
   isExported: false,
 };
 
+// -----------------------------------------------------------------------------
+// FREQUENCY
+// -----------------------------------------------------------------------------
+export const FREQUENCY: AddFunctionDescription = {
+  description: _lt("Calculates the frequency distribution of a range."),
+  args: [
+    arg("data (range<any>)", _lt("The array of ranges containing the values to be counted.")),
+    arg("classes (number, range<any>)", _lt("The range containing the set of classes.")),
+  ],
+  returns: ["RANGE<NUMBER>"],
+  compute: function (data: MatrixArgValue, classes: MatrixArgValue): CellValue[][] {
+    const _data = flattenRowFirst([data], (val) => val)
+      .filter((val) => typeof val === "number" || (typeof val === "string" && isNumber(val)))
+      .map(toNumber);
+    const _classes = flattenRowFirst([classes], (val) => val)
+      .filter((val) => typeof val === "number" || (typeof val === "string" && isNumber(val)))
+      .map(toNumber);
+    /**
+     * Returns the frequency distribution of the data in the classes, ie. the number of elements in the range
+     * between each classes.
+     *
+     * For example:
+     * - data = [1, 2, 3, 4, 5]
```

Comment:
In the example, you can use unsorted value for data
This will be more intuitive to understand that FREQUENCY don't need data in a sorted order to work correctly

Maybe a test for that could be a good idea :)